newrelic/resource_newrelic_nrql_alert_condition.go

```diff
@@ -32,9 +32,9 @@ func termSchema() *schema.Resource {
 			"operator": {
 				Type:         schema.TypeString,
 				Optional:     true,
-				Default:      "equal",
+				Default:      "equals",
 				Description:  "One of (above, below, equal). Defaults to 'equal'.",
-				ValidateFunc: validation.StringInSlice([]string{"above", "below", "equal"}, false),
+				ValidateFunc: validation.StringInSlice([]string{"above", "below", "equals"}, false),
```

Comment:
Thoughts on ignoring casing sensitivity here? I think we've been doing that in most places to make things easier on the user.

Reply:
Good suggestion.  It looks like we already `ToUpper` in the expand, so including a `DiffSuppressFunc` here seems to do the trick.  I'll add that and push it up.